mobilecoind/src/config.rs

```diff
@@ -316,6 +328,31 @@ impl PeersConfig {
             .collect()
     }
 
+    /// Create a consensus enclave verifier, which allows the baked-in
+    /// sigstruct, and any sigstructs produced at run-time via the
+    /// MC_CONSENSUS_ENCLAVE_CSS list. This uses MRENCLAVE verification.
+    pub fn create_consensus_verifier(&self, logger: &Logger) -> Verifier {
```

Comment:
this is probably the most sensitive part of the change -- it changes how mobilecoind is constructing its consensus enclave verifier